.gitignore

```diff
@@ -50,3 +50,6 @@ node_modules/
 *.log
 # Desktop Service Store
 *.DS_Store
+
+#Python virtual environment
+bb_bug/
```

Comment:
Could you please drop unrelated changes?

Reply:
Okay I'll drop the unrelated changes

Reply:
Thank you, @WhiteWolf47!

Reply:
@suhaibmujahid is this alright now?